### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by Vaydheesh**
Solves
- #68 
- #70 added: `apertium-anaphora`
- #71 added: `install_requires` entry in `setup.py`
- #72 allow users to call `apertium.windows_update_path()` irrespective of platform and `raise InstallationNotSupported` if required environment variables aren't present
- added unit tests to increase code coverage to 90%

### CODE REVIEW

**Comment by sushain97** [comment]
Build is failing?

Also, please bump version to `0.2.0`.

**Comment by sushain97** [comment]
Hmm, build still seems to be failing?

**Comment by Vaydheesh** [comment]
installation from apt-get keeps on failing https://travis-ci.com/apertium/apertium-python/jobs/231518743#L313

**Comment by sushain97** [comment]
weird...

File: docs/source/install.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ From PyPI
 
 - Install from PyPI by running
 ```
-pip install apertium-python
+pip install apertium
```
Comment (sushain97) [inline docs/source/install.md]: Are we duplicating these docs??? That is not good...

**Comment by sushain97** [comment]
Released this on pip.

**Pull request merged**